TestApplications/src/Notifier/Form1.cs

```diff
@@ -3,19 +3,24 @@
 
 namespace NotifierTest
 {
-    public partial class Notifier : Form
+    public sealed partial class Notifier : Form
     {
-        public Notifier()
+        public Notifier(string title = "Notifier", bool disableTimer = false)
         {
             InitializeComponent();
 
-            var aTimer = new Timer
+            Text = title;
```

Comment:
where or what for is this text variable used?

Reply:
ok i see this should represent the title of the app

Reply:
The test will change the title bar from the application :)